anyway/flask_app.py

```diff
@@ -153,13 +154,15 @@ def generate_csv(results):
 
 
 def get_kwargs():
-    kwargs = {arg: arg_type(request.values.get(arg, default_value)) for (arg, (arg_type, default_value)) in
-              iteritems(ARG_TYPES)}
+    kwargs = {arg: arg_type(request.values.get(arg, default_value)) for
+              (arg, (arg_type, default_value)) in
+              ARG_TYPES.items()}
     if request.values.get('age_groups[]') == '1234' or request.values.get('age_groups') == '1234':
         kwargs['age_groups'] = '1,2,3,4'
     try:
         kwargs.update(
-            {arg: datetime.date.fromtimestamp(int(request.values[arg])) for arg in ('start_date', 'end_date')})
+            {arg: datetime.date.fromtimestamp(int(request.values[arg])) for arg in
+             ('start_date', 'end_date')})
```

Comment:
This is unrelated to six, I assume. I think it's best not to add unnecessary diffs; if at all, cosmetic changes should be added after the approval.